README.md

```diff
@@ -50,9 +50,21 @@ dsrm.auth.jwt.RefreshExpirationMs = # refresh token expiration time in milisecon
 ./mvnw spring-boot:run
 ```
 
-
 **NOTE:** Running through Maven may require to set `JAVA_HOME` system variable to your JDK location
 
+### Test data
+#### Adding 
+There is a file with SQL inserts `data.sql`
```

Comment:
I would remove this section. I believe it is not needed. It is more technical input. Our readme should describe only how to run and configure application.